frame/clearing-house/src/types.rs

```diff
@@ -13,11 +13,14 @@ pub const BASIS_POINT_DENOMINATOR: u32 = 10_000;
 /// Indicates the direction of a position
 #[derive(Encode, Decode, TypeInfo, Debug, Clone, Copy, PartialEq, Eq)]
 pub enum Direction {
+	/// For a long position, the position is long the asset.
```

Comment:
'For a long position, the *direction* is long the asset.' ? (same for *short*)

(Not sure actually, so just disregard if it's not applicable.)

Reply:
Yeah, the whole thing is a bit redundant, will consider changing it in the next PR